main.tf

```diff
@@ -153,6 +161,32 @@ resource "aws_iam_role_policy" "bastion_host_role_policy" {
 EOF
 }
 
+resource "aws_iam_role_policy" "read_onelogin_keys" {
+  role = aws_iam_role.bastion_host_role.id
+
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ssm:DescribeParameters"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
+                "ssm:GetParameter"
+            ],
+            "Resource": "arn:aws:ssm:${var.region}:*:parameter/bastion/*"
```

Comment:
Does the account need to be `*`? Im wondering if it should be locked to a single account

Reply:
The module doesn't have an explicit account number passed to it.  And other IAM policies (e.g. s3) in the module do not explicitly specify an account number.  Limiting those policies to a specific account could break the module for certain use cases.  It's worth noting that for cross-account permissions to work (both s3 and param store), you'd need the other account to have a reciprocal policy.  So this seems pretty safe to me.

Reply:
@terrancej still looking for any changes here?